Multilingual semantic search (#334)
vespa-engine/blog: _posts/2023-07-25-simplify-search-with-multilingual-embeddings.md

--- 
layout: post
title: "Simplify Search with Multilingual Embedding Models" 
author: jobergum 
date: '2023-07-24' 
image: assets/2023-07-25-simplify-search-with-multilingual-embeddings/bruno-martins-4cwf-iW6I1Q-unsplash.jpg
skipimage: true 
tags: [] 
excerpt: This blog post presents and shows how to represent a robust multilingual embedding model of the E5 family in Vespa. 
---

![Decorative image](/assets/2023-07-25-simplify-search-with-multilingual-embeddings/bruno-martins-4cwf-iW6I1Q-unsplash.jpg)
<p class="image-credit"> Photo by <a
href="https://unsplash.com/@brunus?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Bruno
Martins</a> on <a
href="https://unsplash.com/photos/4cwf-iW6I1Q?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a>
</p>

This blog post presents and shows how to represent a robust
multilingual embedding model of the E5 family in Vespa. We also
demonstrate how to evaluate the model's effectiveness on multilingual
information retrieval (IR) datasets.

## Introduction

The fundamental concept behind embedding models is transforming
textual data into a continuous vector space, wherein similar items

> are brought closer together. In contrast, dissimilar ones are pushed

Reviewer: ```suggestion
are brought close together and dissimilar ones are pushed
```
